README.md

```diff
@@ -1,11 +1,203 @@
-# TIAGo robot
+# TIAGo Robot - Overview
 
-This package contains the description, controllers and bringup for all possible TIAGo configurations (end effectors, force torque sensors..).
+The `tiago_robot` repository contains essential resources for working with PAL Robotics' TIAGo robot, including its URDF files, controllers, and other configurations required for simulation and real-world applications.
 
-To make maintenance easier, the `tiago.urdf.xacro` takes arguments that specify whether the robot has an arm or not, the end effector type, force torque sensor, laser model and many other parameters.
+## About TIAGo
 
-The other configuration files that differ between robot configurations are generated from template files.
+[TIAGo (Take It and Go)](https://pal-robotics.com/robot/tiago/) is a mobile manipulator robot developed by PAL Robotics. It is widely used in research and industry for a variety of applications, including:
 
-The templates are written using [empy](https://pypi.org/project/empy/) and have the extension `.em`. 
+- Mobile manipulation
+    
+- Human-robot interaction
+    
+- Assistive robotics
+    
+- Warehouse automation
+    
+- AI and reinforcement learning applications
+    
 
-To regenerate a group of files, you must execute `ros2 run tiago_bringup regen_em_file.py EM_FILE_NAME` from the directory where the `.em` file is.
+## Features of this Repository
+
+This repository includes:
+
+- **URDF/Xacro files**: Detailed robot description models defining TIAGo’s structure and kinematics.
+    
+- **ROS 2 configuration files**: Launch files, controllers, and sensor integrations for seamless ROS 2-based operation.
+    
+- **Simulation support**: Compatibility with Gazebo and MoveIt for motion planning and testing.
+    
+- **Hardware integration**: Configuration files for real-world deployments.
+    
+
+## Installation & Setup
+
+### Prerequisites
+
+Ensure that you have the following installed:
+
+- ROS 2 Humble
+    
+- Gazebo (for simulation support)
+    
+- MoveIt 2 (for motion planning)
+    
+- PAL Robotics dependencies
+    
+- Git (to clone the repository)
+    
+
+### Cloning the Repository
+
+```bash
+cd ~/tiago_ws/src  # Navigate to your ROS 2 workspace
+git clone https://github.com/pal-robotics/tiago_robot.git
+cd ~/tiago_ws && colcon build
+```
+
+### Setting Up the Environment
+
+Before using the TIAGo robot, source the necessary setup files:
+
+```bash
+source ~/tiago_ws/install/setup.bash
+```
+
+To avoid sourcing manually every time, add the above line to your `.bashrc` file.
```

Comment:
I would simply add the binary installation, `ros-humble-tiago-robot`, this should install all the dependencies etc